Implement "Set Up System Files" on Android

PabloMK7 opened the pull request:
Implements the neccessary UI changes so that it's possible to perform the system files setup.

PabloMK7 committed:
Implement "Set Up System Files" on Android

OpenSauce04 committed:
Use correct strings + Remove chunks of unused code

OpenSauce04 commented on src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt:
```diff
@@ -121,8 +121,8 @@ class HomeSettingsFragment : Fragment() {
+                R.string.setup_system_files,
+                R.string.setup_system_files_description,
                 R.drawable.ic_system_update,
                 {
```
We should probably move this setting underneath "Install CIA File". I find it unlikely that people will be setting up their system files as often as they'll be installing applications.

OpenSauce04 committed:
Updated license header

OpenSauce04 commented on src/android/app/src/main/res/values/strings.xml:
```diff
@@ -142,28 +144,21 @@
-    <string name="cancelling">Cancelling…</string>
     <string name="home_menu">HOME Menu</string>
     <string name="home_menu_warning">System Files Warning</string>
     <string name="home_menu_warning_description">Due to how slow Android\'s storage access framework is for accessing Azahar\'s files, downloading multiple versions of system files can dramatically slow down loading for applications, save states, and the Applications list. Only download the files that you require to avoid any issues with loading speeds.</string>
```
This message needs to be updated.

OpenSauce04 committed:
SystemFilesFragment.kt: Use radio buttons for selecting O3DS/N3DS

OpenSauce04 committed:
HomeSettingsFragment.kt: Moved `Install CIA` above `Set Up System Files`

OpenSauce04 committed:
strings.xml: Updated system file setup button strings

OpenSauce04 committed:
android: Remove System Files Warning

This warning is no longer relevant due to changes in how system files are installed
